src/functions/is-labeled.ts

```diff
@@ -21,6 +21,6 @@ export function isLabeled(
   });
 }
 
-function cleanLabel(label: Readonly<string>): CleanLabel {
+export function cleanLabel(label: Readonly<string>): CleanLabel {
```

Comment:
When I started to reduce the complexity of the main class, I decided to go for one export per file regarding most of the code, including the functions.
Dunno WDYT of this subtle detail, but for consistency I would prefer a dedicated file for this function.